.travis.yml

```diff
@@ -48,7 +48,7 @@ env:
         # that tox passes through to the {posargs} indicator in tox.ini.
         # The latter can be used for example to pass arguments to pytest.
         - TOXENV='test'
-        - TOXARGS='-v'
+        - TOXARGS='-rv'
```

Comment:
@astrofrog and @Cadair , is this where I am supposed to apply `-r`? If it is, didn't help.